www.foia.gov/_sass/_base.scss

```diff
@@ -1 +1,12 @@
 @import 'mixins/text_divider';
+.print-hide {
+  @media print {
```

Comment:
I think this should probably end up broken out into a `_print.scss` partial that just is one big `@media print` query, but it's fine for now.